sphinx/quickstart.py

```diff
@@ -447,9 +454,11 @@ def write_file(fpath, content, newline=None):
     if d.get('make_mode') is True:
         makefile_template = 'quickstart/Makefile.new_t'
         batchfile_template = 'quickstart/make.bat.new_t'
+        poshfile_template = 'quickstart/make.ps1_t'
```

Comment:
(required)
Since sphinx-1.5, we will provide `make-mode` version Makefile/make.bat (.new_t) by default.
`make-mode` doesn't include make target names in the Makefile/make.bat file.

`make.ps1_t` looks like old-style Makefile/make.bat, so can you provide `make_mode` version `make.ps1.new_t` file?